secrets/dataplane-observability-config.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# the url points to an auth proxy before the remote write endpoint
+# this is optional, either proxy or direct configurations must be provided
+proxy:
+  remoteWriteUrl: https://observability-rw-proxy-managed-services-stage.apps.app-sre-stage-0.k3s7.p1.openshiftapps.com
+
+# the url points directly to an (authenticated) remote write endpoint
+# this is optional, either proxy or direct configurations must be provided
+# direct:
+#   remoteWriteUrl: https://observatorium-mst.api.stage.openshift.com
+#   metricsClientId: dummy
+#   metricsClientSecret: dummy
```

Comment:
What about the dex based support? We could also consider adding them in this file. Or we could scope that to a future improvement once this initial structure support is in

Reply:
If we want to continue supporting dex based Observatorium, we can consider moving the config into this file. Although it looks quite different, there is a long lived token instead of clientId and secret.